ansible/deploy-prometheus-grafana.yml

```diff
@@ -39,7 +39,7 @@
         static_configs:
           - targets: "{{ groups['redpanda'] | map('extract', hostvars, 'inventory_hostname') | map('regex_replace', '^(.*)$','\\1:9644') | list  }}"
       - job_name: "node"
-        metrics_path: "/public_metrics"
+        metrics_path: "/metrics"
```

Comment:
Why this change?
/public_metrics is meant to be for public consumption
/metrics is far too detailed

Reply:
```    prometheus_scrape_configs:
      - job_name: "redpanda"
        metrics_path: "/public_metrics"
        static_configs:
          - targets: "{{ groups['redpanda'] | map('extract', hostvars, 'inventory_hostname') | map('regex_replace', '^(.*)$','\\1:9644') | list  }}"
      - job_name: "node"
        metrics_path: "/public_metrics"
        metrics_path: "/metrics"```

Node is the scrape for nodeexporter, we're still scraping public_metrics for redpanda.

Reply:
Yes, see line 38